src/main/java/duke/Parser.java

```diff
@@ -0,0 +1,105 @@
+package duke;
+
+
+import exceptions.DukeException;
+import tasks.*;
```

Comment:
you can replace the wildcard import with list of class you are using here. According to our coding standards, imported classes should always be listed explicitly. See [Coding Standards](https://se-education.org/guides/conventions/java/basic.html#statements)